index.html

```diff
@@ -119,100 +119,292 @@ <h3 class="work-name">Tonic</h3>
               <li class="lang-section">css</li>
               <li class="lang-section">javascript</li>
             </ul>
-            <button type="button" class="work-btn">See Project</button>
+            <button type="button" data-modal-target="#modal1" class="work-btn">
+              See Project
+            </button>
           </div>
-        </li>
+          <div class="modal" id="modal1">
+            <div class="modal-header">
+              <div class="title">Tonic</div>
+
+              <button type="button" data-close-button class="close-button">&times;</button>
```

Comment:
- [ ] When a user clicks the `X` button on the popup window, the window should disappear

Reply:
Well resolved this!